### ISSUE TITLE
More sample data may be required to try stitching surfaces in STEP3

### ISSUE THREAD

**Issue by EVANGELION-Air** [MultiDIC/MultiDIC#34]
The current example (in v1.1.0) only have image data from two adjacent cameras (as a camera pair), which means that we can only implement one surface reconstruction from this only one camera pair, and completely unable to attempt stitching of multiple surfaces  STEP3. I think this is one of the most critical content in author's paper. 

In brief, I would be very appreciate if author could provide more sampled image data. 

I understand this may be time consuming for the author, but i think three adjacent cameras (i.e two adjacent camera pairs) is good enough to try stitching surfaces, which means upload image data obtain by another adjacent camera (maybe camera number 200 or 202) is sufficient.

**Comment by danasolav** [comment]
I will try to add data with more than two cameras soon.

**Comment by danasolav** [comment]
That's correct. Stitching cannot be currently tested using the sample data.
I will add images from another camera soon.
